BookingsApi/BookingsApi.DAL/Helper/CloneHearingToCommandMapper.cs

```diff
@@ -80,7 +80,7 @@ public static class CloneHearingToCommandMapper
 
         private static List<LinkedParticipantDto> GetLinkedParticipantDtos(Hearing hearing)
         {
-            var hearingParticipants = hearing.Participants.Where(x => x.LinkedParticipants.Any()).ToList();
+            var hearingParticipants = hearing.Participants.Where(x => x.HearingRole.Name.ToLower() == "interpreter" &&  x.LinkedParticipants.Any()).ToList();
```

Comment:
If the HearingRole = 'interpreter', then we don't need `x.LinkedParticipants.Any()`because all interpreters will be linked participants and will have at least one anyway